docs/docsite/rst/inventory_guide_iocage_aliases.rst

```diff
@@ -0,0 +1,208 @@
+..
+  Copyright (c) Ansible Project
+  GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+  SPDX-License-Identifier: GPL-3.0-or-later
+
+.. _ansible_collections.community.general.docsite.inventory_guide.inventoy_guide_iocage.aliases:
+
+Aliases
+^^^^^^^
+
+Quoting :ref:`inventory_aliases`
+
+   The inventory_hostname is the unique identifier for a host in Ansible, this can be an IP or a hostname, but also just an 'alias' or short name for the host.
```

Comment:
```suggestion
   The ``inventory_hostname`` is the unique identifier for a host in Ansible, this can be an IP or a hostname, but also just an 'alias' or short name for the host.
```